solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -1979,6 +1981,7 @@ public ScoreMode scoreMode() {
       // no docs on this page, so cursor doesn't change
       qr.setNextCursorMark(cmd.getCursorMark());
     } else {
+      fullSortCount.increment();
```

Comment:
Oh I found a better place for this!  Just one place :-).  `buildTopDocsCollector`.  Then you can remove these 3.

Reply:
addressed in 0844a7a